<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Asset/UI Home.png);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseTemplate">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(65, 141, 237, 255), stop:1 rgba(22, 55, 229, 255));
    color: white;
    border-radius: 10px;
	font-size: 20px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: normal; 
    font-style: normal;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(15, 30, 180, 255), stop:1 rgba(50, 100, 200, 255));
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Template">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
border-bottom: 2px solid rgba(255, 255, 255, 100);
padding-bottom:7px;
font-size: 15px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: bold; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Template File Location</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StartColor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
border-bottom: 2px solid rgba(255, 255, 255, 100);
padding-bottom:7px;
font-size: 15px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: 100; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Start Color (Hex)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EndColor">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
border-bottom: 2px solid rgba(255, 255, 255, 100);
padding-bottom:7px;
font-size: 15px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: 100; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>End Color (Hex)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="BorderSize">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
border-bottom: 2px solid rgba(255, 255, 255, 100);
padding-bottom:7px;
font-size: 15px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: 100; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Border Size (px)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 47, 47, 255));
    color: white;
    border-radius: 0px;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
font-size: 30px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: 100; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string>Border Setting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
font-size: 30px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: 100; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string>Upload The Template</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
border-bottom: 2px solid rgba(255, 255, 255, 100);
padding-bottom:7px;
font-size: 15px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: bold; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter Saving Folder Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseSaving">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(65, 141, 237, 255), stop:1 rgba(22, 55, 229, 255));
    color: white;
    border-radius: 10px;
	font-size: 20px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: normal; 
    font-style: normal;
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(15, 30, 180, 255), stop:1 rgba(50, 100, 200, 255));
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgb(255, 255, 255);
font-size: 30px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: 100; 
    font-style: normal;</string>
   </property>
   <property name="text">
    <string>Saving Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>491</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SF Pro Display</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(65, 141, 237, 255), stop:1 rgba(22, 55, 229, 255));
    color: white;
    border-radius: 10px;
	font-size: 30px;
font-family: &quot;SF Pro Display&quot;; 
	font-weight: bold; 
    font-style: normal;
}


QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(15, 30, 180, 255), stop:1 rgba(50, 100, 200, 255));
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Asset/Rsc.qrc"/>
 </resources>
 <connections/>
</ui>
